hack/go-vet.sh

```diff
@@ -24,9 +24,9 @@ if [[ -z ${CI} ]]; then
     which ./bin/openapi-gen > /dev/null || go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
     # generate the openapi files
     echo "Generating openapi files"
-    ./bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/app/v1alpha1 -O zz_generated.openapi -p ./pkg/apis/app/v1alpha1 -h ./hack/boilerplate.go.txt -r "-"
+    ./bin/openapi-gen --logtostderr=true -o "" -i github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1 -O zz_generated.openapi -p ./pkg/apis/app/v1alpha1 -h ./hack/boilerplate.go.txt -r "-"
     ./bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/kafka/v1beta1 -O zz_generated.openapi -p ./pkg/apis/kafka/v1beta1 -h ./hack/boilerplate.go.txt -r "-"
```

Comment:
why don't you do it also here ?

Reply:
+1

Reply:
That's true, the same could be done here. I wonder why this didn't produce any changes to the file during my testing though, even after modifying this on line 28 as well. I think it's worth changing anyway. I'm pushing a new commit with this.